src/ansible_navigator/actions/_actions.py

```diff
@@ -16,7 +16,7 @@
 try:
     from importlib import resources
 except ImportError:
-    import importlib_resources as resources  # type: ignore
+    import importlib_resources as resources  # type: ignore[import, no-redef]
```

Comment:
One of the idioms to use with mypy is if-else rather than try-except for fallback imports. It's a known bug that mypy does not understand that it's one or the other. But with the `if py_ver:` style, it is able to figure out what's going on and wouldn't need ignores.

Reply:
@webknjaz  Good idea, although I think we should wait on this until we get better python version test coverage in place when CI is broght up to speed.  I'll log an issue so it can be changed later.

Reply:
Related: #646